recipe_scrapers/monsieurcuisine.py

```diff
@@ -1,72 +1,122 @@
 # mypy: disallow_untyped_defs=False
-from ._abstract import AbstractScraper
-from ._utils import get_minutes, get_yields, normalize_string
+import re
+
+import requests
+
+from ._abstract import HEADERS, AbstractScraper
+from ._utils import get_yields
+
+SCRIPT_PATTERN = re.compile(r'"recipeId":(\d+)')
 
 
 class MonsieurCuisine(AbstractScraper):
+    def __init__(self, url, proxies=None, timeout=None, *args, **kwargs):
+        super().__init__(url=url, proxies=proxies, timeout=timeout, *args, **kwargs)
+        recipe_id = None
+        for script in self.soup.find_all("script"):
+            matches = SCRIPT_PATTERN.search(str(script.string))
+            if matches:
+                recipe_id = matches.group(1)
+        language_iso = self.soup.find("html")["lang"]
```

Comment:
Let's re-use some existing code here:

```suggestion
        language_iso = self.language()
```